os/drivers/lcd/lcd_dev.c

```diff
@@ -356,6 +399,19 @@ int lcddev_register(struct lcd_dev_s *dev)
 
 	priv->dev = dev;
 
+	sem_init(&flushing_sem, 0, 0);
+
+	g_wait = false;
+	empty = true;
+
+	lcd_kbuffer = (uint8_t *)kmm_malloc(CONFIG_LCD_XRES * CONFIG_LCD_YRES * sizeof(uint16_t));
+
+	int pid = kernel_thread("flushing", 204, 8192, lcd_flushing_thread, NULL);
```

Comment:
Based on the current code, if multiple LCDs are init, thread creation will fail from the second init.

Reply:
And here it seems better to deliver struct lcd_dev_s *dev to the argument of the kernel thread.